test/unit/Transformers/JsonApi/responseSerializer.test.js

```diff
@@ -0,0 +1,88 @@
+import { matchers } from 'jest-json-schema';
+import responseNormalizer from '../../../../src/Transformers/JsonApi/responseNormalizer';
+import jsonResponseFixture from '../../../fixtures/fetchJsonResponse';
+import resourceJsonSchema from '../../../../schema/resource.schema';
+
+expect.extend(matchers);
+
+describe('Validations for the responseNormalizer', () => {
+  test('that the responseNormalizer returns a schema-valid array of resources', () => {
+    const result = responseNormalizer(jsonResponseFixture);
+
+    expect(result).toHaveLength(2);
+
+    expect(resourceJsonSchema).toBeValidSchema();
```

Comment:
Ik zou dit los trekken. Eerst het schema testen en dan testen of de normalizer volgens het schema iets teruggeeft.